lib/http.js

```diff
@@ -998,8 +999,8 @@ OutgoingRequest.prototype._start = function _start(stream, options) {
   this.headersSent = true;
 
   this.emit('socket', this.stream);
-
   var response = new IncomingResponse(this.stream);
+  response.req = this;
```

Comment:
I picked response.req because that is the same as the node HTTP library.  https://github.com/joyent/node/blob/master/lib/_http_client.js#L423 

I am not a huge fan on needlessly short name either, but I would promote compatibility with the core HTTP.